core/common/src/main/java/alluxio/client/file/CacheContext.java

```diff
@@ -24,6 +24,9 @@
  * Cache related context.
  */
 public class CacheContext {
+
+  public static final CacheContext DEFAULT_CACHE_CONTEXT = new CacheContext();
```

Comment:
I removed the singleton default "DEFAULT_CACHE_CONTEXT" and use the defaults() method instead, because the cacheContext is mutable, so if somewhere modify the DEFAULT_CACHE_CONTEXT, it would cause unexpected behaviors in other places.   What do you think?

Reply:
yes, it is the case I am concerning too.